themes/cloud/v1/theme.json

```diff
@@ -0,0 +1,194 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetwriters/notesnook-themes/main/schemas/v1.schema.json",
+  "name": "Cloud",
+  "id": "cloud",
+  "version": 1,
+  "license": "GPL-3.0-or-later",
+  "homepage": "https://notesnook.com",
+  "description": "A clear environment where focus remains",
+  "colorScheme": "light",
+  "compatibilityVersion": 1,
+  "authors": [
+    {
+      "name": "Kenneth V.",
+      "email": "support@streetwriters.co",
+      "url": "https://streetwriters.co"
```

Comment:
```suggestion
      "name": "Kenneth V."
```

Reply:
Hey, I am not really seeing any change here.

Reply:
Ok, thank you for pointing this out. Will change it right away.